substrate/bin/node/runtime/src/lib.rs

```diff
@@ -1573,6 +1578,30 @@ impl pallet_mmr::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	/// Version of the produced MMR leaf.
+	///
+	/// The version consists of two parts;
+	/// - `major` (3 bits)
+	/// - `minor` (5 bits)
+	///
+	/// `major` should be updated only if decoding the previous MMR Leaf format from the payload
+	/// is not possible (i.e. backward incompatible change).
+	/// `minor` should be updated if fields are added to the previous MMR Leaf, which given SCALE
+	/// encoding does not prevent old leafs from being decoded.
+	///
+	/// Hence we expect `major` to be changed really rarely (think never).
+	/// See [`MmrLeafVersion`] type documentation for more details.
```

Comment:
Why is this documentation copied everywhere? What is the purpose? This should be above the `MmrLeafVersion` type, if this isn't there yet.

Reply:
Sorry, I copied it blindly. Fixed.